docs/query-builder.md

```diff
@@ -448,64 +448,126 @@ DB::collection('items')
 
 **Push**
 
-Add items to an array.
+Add one or multiple values to the items array.
 
 ```php
+// Push the value to the matched documents
 DB::collection('users')
-    ->where('name', 'John')
-    ->push('items', 'boots');
+  ->where('name', 'John')
+    // Push a single value to the items array
+  ->push('items', 'boots'); // items: ['boots']
+
+DB::collection('users')
+  ->where('name', 'John')
+    // Push multiple values to the items array
+  ->push('items', ['hat', 'jeans']); // items: ['boots', 'hat', 'jeans']
 
+// Or
+
+// Push the value to an exact document
 $user->push('items', 'boots');
+$user->push('items', ['hat', 'jeans']);
 ```
 
+To add an array to the messages array.
```

Comment:
We should add a note that values arrays must be nested into a list array, otherwise they are interpreted as a list of scalar values.